app/views/career_site/positions/index.html.slim

```diff
@@ -6,7 +6,7 @@
   .list-group.gap-3 id="position-list" role="list"
     - @positions.each do |position|
       .list-group-item.custom-list-item.px-1 id="position_#{position.id}"
-        .position_name.text-break.mb-1
-          = link_to position.name,
-                    career_site_position_path(tenant_slug: @current_tenant.slug, id: position.slug)
-        .position_location.mb-0 = position&.location&.short_name
+        = link_to career_site_position_path(tenant_slug: @current_tenant.slug, id: position.slug),
+                  class: "position-link" do
```

Comment:
for what do we need this class?

Reply:
It was needed for css-customization of the site. But in the current implementation it is not necessary. You are right)